pontoon/base/views.py

```diff
@@ -418,7 +418,9 @@ def get_translation_history(request):
                 "date": t.date.strftime("%b %d, %Y %H:%M"),
                 "date_iso": t.date.isoformat(),
```

Comment:
We no longer use `translation.date` on frontend, so we can drop it. We can also drop `.isoformat()`. At that point we should rename `date_iso` to `date` (also on the frontend).

```suggestion
                "date": t.date,
```